src/IIIFPresentation/API.Tests/Integration/GetCollectionTests.cs

```diff
@@ -378,11 +379,11 @@ public async Task Get_ChildFlat_IgnoresOrderBy_WhenCalledWithInvalidOrderBy()
         var collection = await response.ReadAsPresentationJsonAsync<PresentationCollection>();
         
         // Assert
-        collection.TotalItems.Should().Be(2);
+        collection.TotalItems.Should().Be(3);
```

Comment:
Would it be worth defining number of items belonging to root somewhere as a constant variable?